lib/services/suite/suite.go

```diff
@@ -1140,6 +1141,8 @@ func CollectOptions(opts ...Option) Options {
 
 // ClusterConfig tests cluster configuration
 func (s *ServicesTestSuite) ClusterConfig(c *check.C, opts ...Option) {
+	ctx := context.Background()
```

Comment:
Since you're adding a `ctx` variable here, could you replace all the `context.TODO` occurrences in this function with `ctx` for the sake of consistency?